src/parser.ts

```diff
@@ -77,13 +72,16 @@ export type ComponentNameResolver = (
   source: ts.SourceFile
 ) => string | undefined | null | false;
 
+type SortFunction = (prop: PropItem[]) => PropItem[];
```

Comment:
I am not sure if this is the best function definition for sorting. I mean, this function can also filter items out or modify them in any way. This doesn't need to be bad but it seems more then `sort` to me. 
Maybe it could have a different name (I cannot think of any). Or could be changed to something like `sort: (a, b) => number`.